output-multi.cpp

```diff
@@ -364,7 +364,7 @@ int output_multi_t::process_relation(osmid_t id, const memberlist_t &members,
         //all this trickery
         int make_boundary, make_polygon = 1;
         taglist_t outtags;
-        filter = m_tagtransform->filter_rel_member_tags(tags, filtered, m_relation_helper.roles,
```

Comment:
this should probably be called with the tags from the relation, not the transformed tags from the relation